src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -22,17 +22,30 @@
 #include <ripple/app/main/Application.h>
 #include <ripple/app/misc/Validations.h>
 #include <ripple/app/data/DatabaseCon.h>
+#include <ripple/core/ConfigSections.h>
 #include <boost/format.hpp>
+#include <boost/tokenizer.hpp>
 
 namespace ripple {
 
+namespace detail
+{
+/** Amendments that are "compiled into" this app
+
+    Add amendments to this collection at build time to enable them on this
+   server. The first element of the pair is the id, the second element is
+   the "friendly name".
+*/
+std::set<std::pair<std::string, std::string>> PreEnabledAmendments;
+}
+
 /** Track the list of "amendments"
 
     An "amendment" is an option that can affect transaction processing
```

Comment:
Lets try it without the tag for now, I'd like to see code that is stuck in the pull request process marked Tx Change get modified to use the amendments.